README.md

```diff
@@ -1,4 +1,75 @@
 _Note that this project depends on internal Symphony infrastructure (repository.symphony.com), and therefore it can only be built by Symphony LLC employees/partners._
 
-# App-Integrations-Universal
-Universal Webhook Integration
+# Universal Webhook Integration
+The Universal Webhook Integration enables you to send messages directly from any service into a Symphony conversation of your choice - either a 1:1 IM with you, or a room that you are the owner of.
+
+## How it works
+If you have a service that can be configured to send webhooks, all you have to do is point it to the URL you generate in the Universal Webhook Integration configuration pane and then format a message in messageML.
+
+## What formats and events it support and what it produces
+Every integration will get a message sent in a specific format (depending on what system is it dealing with) and it will usually convert it into an "entity" before it reaches the Symphony platform.
+It will also, usually, identify the kind of message it will deal with based on an "event" identifier, that varies based on which system is it integrating with.
+
+The Universal Webhook in the other hand does not support any special events, and it merely forwards the message received (if valid). 
+It deals with messages in the `<messageML></messageML>` format only.
+
+The validation it submits the message will enforce the rules presented [here](https://rest-api.symphony.com/docs/message-format/)
+
+Below is an example of:
+* What sort of message you can send through the Universal Webhook Integration
```

Comment:
Instead of making this as bullets, I would make them subsections of a "Sample Universal Webhook Message".